common/module.mk

```diff
@@ -19,6 +19,39 @@ MODULE_OBJS := \
 	json.o \
 	language.o \
 	localization.o \
+	lua/double_serialization.o \
+	lua/lapi.o \
+	lua/lauxlib.o \
+	lua/lbaselib.o \
+	lua/lcode.o \
+	lua/ldblib.o \
+	lua/ldblib.o \
+	lua/ldebug.o \
+	lua/ldo.o \
+	lua/lfunc.o \
+	lua/lgc.o \
+	lua/linit.o \
+	lua/liolib.o \
+	lua/llex.o \
+	lua/lmathlib.o \
+	lua/lmem.o \
+	lua/loadlib.o \
+	lua/lobject.o \
+	lua/lopcodes.o \
+	lua/loslib.o \
+	lua/lparser.o \
+	lua/lstate.o \
+	lua/lstring.o \
+	lua/lstrlib.o \
+	lua/ltable.o \
+	lua/ltablib.o \
+	lua/ltm.o \
+	lua/lua_persist.o \
+	lua/lua_persistence_util.o \
+	lua/lua_unpersist.o \
+	lua/lvm.o \
+	lua/lzio.o \
+	lua/scummvm_file.o \
```

Comment:
These should not be compiled in all the time in common code, which is why the feature flag has been requested. They should be enclosed in a ENABLE_LUA ifdef. i.e:

    ifdef ENABLE_LUA
    MODULE_OBJS += \
	xxx.o
    endif

Check common/module.mk for an example

Reply:
I have wrapped it now